packages/salesforcedx-vscode-lwc/src/testSupport/testRunner/testRunner.ts

```diff
@@ -95,7 +95,7 @@ export class TestRunner {
       runTestsByPathArgs = [];
     }
     const testNamePatternArgs = testName
-      ? ['--testNamePattern', `"${escapeStrForRegex(testName)}"`]
+      ? ['--testNamePattern', `${escapeStrForRegex(testName)}`]
```

Comment:
Any findings about what changed here to make this _stop_ working? 


a jest unit test around this particular logic could give us more clarity about how/why it's written this way.

Reply:
This was more of a "try and find out" and it worked, but I can definitely add a unit test here too